content/issues/1/data-beyond-vision/index.md

```diff
@@ -13,15 +13,15 @@ pdf: https://cdh.princeton.edu/media/uploads/documents/grad_fellows_call_for_spr
 images: ["issues/1/data-beyond-vision/images/dbv-social.jpg"]
 ---
 
-
-How do we represent tangible objects in a visual medium? We use words, pictures, and diagrams. We describe, share, show, and fail.
+How do we represent tangible objects in a visual medium? We use words, pictures, and diagrams. We describe, share, show and fail.
```

Comment:
guess we need a house ruling on oxford commas...

Reply:
I'm usually an Oxford comma guy but think I kept it out of this one because I liked the "show and fail" echo with "show and tell." But I agree, it should be in there.